packages/edit-site/src/components/editor/index.js

```diff
@@ -138,6 +149,11 @@ export default function Editor( { isLoading, onClick } ) {
 		};
 	}, [] );
 
+	const { setSectionRootBlockName } = useDispatch( blocksStore );
+	useEffect( () => {
+		setSectionRootBlockName( getSectionRootBlockName( context?.postType ) );
+	}, [ context, setSectionRootBlockName ] );
```

Comment:
```suggestion
	}, [ context?.postType, setSectionRootBlockName ] );
```

Dep can be simplified to avoid re-running the effect.

Reply:
Removed the effect.